input/fsh/VRCL/harmonized-profiles/ObservationMethodOfDisposition_new.fsh

```diff
@@ -0,0 +1,18 @@
+Profile: ObservationDecedentDispositionMethodVitalRecordsNew
+Parent: Observation
+Id: Observation-decedent-disposition-method-vr-new
+Title: "Decedent Disposition Method"
+Description: "Decedent Disposition Method (Observation)"
+//* status 1..1 loosening this restriction as BFDR does not have status 
+//* status only code
+//* status = #final 
+* code 1..1
+* code only CodeableConcept
+* code = $loinc#80905-3 // "Body disposition method"
+* subject 1..1
+* subject only Reference(Decedent) //BFDR references Patient - Decedent Fetus - Vital Records, which is a USCorePatient. Need to consolidate?
+* performer ^short = "Mortician" //BFDR does not include performer
+* performer ^definition = "Performer is optional, but if provided should be the Mortician using a USCorePractitioner."
```

Comment:
* performer ^definition = "Mortician is optional, and is not referenced by the fetal death or mortality use cases"